conversion.go

```diff
@@ -0,0 +1,32 @@
+package seccomp // import "github.com/seccomp/containers-golang"
+
+import "fmt"
+
+var goArchToSeccompArchMap = map[string]Arch{
+	"386":         ArchX32,
+	"amd64":       ArchX86_64,
+	"amd64p32":    ArchX86,
```

Comment:
Actually. this is swapped - `amd64p32` is x32, `386` is X86

Reply:
Whoops, fixed that thank you! :pray: